src/Sylius/Component/ImportExport/Model/ExportProfile.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\ImportExport\Model;
+
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ExportProfile extends Profile implements ExportProfileInterface
+{
+    const DEFAULT_READER = 'user_orm';
+    const DEFAULT_WRITER = 'csv_writer';
+
+    public function __construct()
+    {
+        $this->reader = ExportProfile::DEFAULT_READER;
```

Comment:
Why we initialize export profile with 'user_orm' reader as default? Component doesn't know anything about users.